interfaces/polkit/backend.go

```diff
@@ -0,0 +1,120 @@
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/timings"
+)
+
+func polkitPolicyName(snapName, nameSuffix string) string {
+	return snap.ScopedSecurityTag(snapName, "interface", nameSuffix) + ".policy"
+}
+
+// Backend is responsible for maintaining DBus policy files.
+type Backend struct{}
+
+// Initialize does nothing.
+func (b *Backend) Initialize(*interfaces.SecurityBackendOptions) error {
+	return nil
+}
+
+// Name returns the name of the backend.
+func (b *Backend) Name() interfaces.SecuritySystem {
+	return interfaces.SecurityPolkit
+}
+
+// Setup installs the polkit policy files specific to a given snap.
+//
+// Polkit has no concept of a complain mode so confinment type is ignored.
+func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions, repo *interfaces.Repository, tm timings.Measurer) error {
+	snapName := snapInfo.InstanceName()
+	// Get the policies that apply to this snap
+	spec, err := repo.SnapSpecification(b.Name(), snapName)
+	if err != nil {
+		return fmt.Errorf("cannot obtain polkit specification for snap %q: %s", snapName, err)
+	}
+
+	// Get the files that this snap should have
+	glob := polkitPolicyName(snapName, "*")
+	content := deriveContent(spec.(*Specification), snapInfo)
+	dir := dirs.SnapPolkitPolicyDir
+	if err := os.MkdirAll(dir, 0755); err != nil {
+		return fmt.Errorf("cannot create directory for polkit policy files %q: %s", dir, err)
+	}
+	_, _, err = osutil.EnsureDirState(dir, glob, content)
+	if err != nil {
+		return fmt.Errorf("cannot synchronize polkit policy files for snap %q: %s", snapName, err)
+	}
+	return nil
+}
+
+// Remove removes dbus configuration files of a given snap.
+//
+// This method should be called after removing a snap.
+func (b *Backend) Remove(snapName string) error {
+	glob := polkitPolicyName(snapName, "*")
+	_, _, err := osutil.EnsureDirState(dirs.SnapPolkitPolicyDir, glob, nil)
+	if err != nil {
+		return fmt.Errorf("cannot synchronize polkit files for snap %q: %s", snapName, err)
+	}
+	return nil
+}
+
+// deriveContent combines security snippets collected from all the interfaces
+// affecting a given snap into a content map applicable to EnsureDirState.
+func deriveContent(spec *Specification, snapInfo *snap.Info) map[string]osutil.FileState {
+	policies := spec.Policies()
+	if len(policies) == 0 {
+		return nil
+	}
+	content := make(map[string]osutil.FileState, len(policies)+1)
+	for nameSuffix, policyContent := range policies {
+		filename := polkitPolicyName(snapInfo.InstanceName(), nameSuffix)
+		content[filename] = &osutil.MemoryFileState{
+			Content: policyContent,
+			Mode:    0644,
+		}
+	}
+	return content
+}
+
+func (b *Backend) NewSpecification() interfaces.Specification {
+	return &Specification{}
+}
+
+// SandboxFeatures returns list of features supported by snapd for dbus communication.
```

Comment:
```suggestion
// SandboxFeatures returns nil
```
to follow similar empty implementations of other backends.